lfs/batcher_test.go

```diff
@@ -7,28 +7,66 @@ import (
 )
 
 func TestBatcherSizeMet(t *testing.T) {
-	b := NewBatcher(2)
+	assertAll([]batcherTestCase{
+		{2, 4, false},
+		{3, 5, false},
+		{0, 0, false},
+	}, t)
+}
+
+func TestBatcherExit(t *testing.T) {
+	assertAll([]batcherTestCase{
+		{2, 4, true},
+		{3, 5, true},
+		{0, 0, true},
+	}, t)
+}
+
+// Type batcherTestCase specifies information about how to run a particular test
+// around the type lfs.Batcher.
+type batcherTestCase struct {
```

Comment:
Go comments for types and funcs start with the name. The only godoc comments that have a prefix are package comments.  https://golang.org/doc/effective_go.html#commentary